source/scale/high-availability-cluster.rst

```diff
@@ -314,6 +314,22 @@ The following configuration was tested on an AWS Aurora r5.xlarge instance of Po
 
 8. **tcp_keepalives_interval**: 1
 
+9. If you have beefy server with more than 32 CPUs, please set these options to utilize more CPU for your server:
```

Comment:
Is "beefy" a type of server or a reference to the level of power?

Reply:
The latter. Let me know if there's a better word I should use :stuck_out_tongue:

Reply:
It's a good word! I propose using something like high-spec in this case, for ease of understanding. What are your thoughts?

Reply:
Sounds good to me!